_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -0,0 +1,438 @@
+
+			<a href="http://twitter.com">
+				<text x="50" y="90" text-anchor="middle">
+					Follow us
+				</text>
+			</a>
+		</svg>
+	</p>
+</html>
+```
+
+#### Failed Example 6
+
+Both links resolve to [same resource][] after redirect, but the redirect is not instant.
+
+```html
+<html lang="en">
+	<p>
+		Learn more (<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html"
+			>Contact us</a
+		>) and get in touch (<a
+			href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/redirect1.html"
+			>Contact us</a
+		>)
+	</p>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+These `a` and `area` elements do not have a role of `link`.
+
+```html
+<html lang="en">
+	<a>Link text</a>
+	<area aria-label="Link text" />
+</html>
+```
+
+#### Inapplicable Example 2
+
+These links have different [accessible names][accessible name]. The rule only applies to identical [accessible names][accessible name], not to identical link destinations.
+
+**Note:** It is a best practice for [Success Criterion 2.4.4: Link Purpose (In Context)][sc244] that identical links have identical [accessible names][accessible name]. This is however not a requirement.
+
+```html
+<html lang="en">
+	<p>
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html">Reach out</a>
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html">Contact us</a>
+	</p>
+</html>
+```
+
+#### Inapplicable Example 3
+
+These links have the same [accessible name][] and link to the [same resource][] but different [programmatically determined link contexts][programmatically determined link context].
+
+```html
+<html lang="en">
+	<ul>
+		<li>
+			To learn more about us:
+			<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html">Contact us</a>
+		</li>
+		<li>
+			To get in touch with us:
+			<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html">Contact us</a>
+		</li>
+	</ul>
+</html>
+```
+
+#### Inapplicable Example 4
+
+These `span` elements do not have a [semantic role][] of link.
+
+```html
+<html lang="en">
+	<p>
+		Learn more (<span
+			onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/page1.html'"
+		>
+			Contact Us </span
+		>) and get in touch (<span
+			onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/page2.html'"
+		>
+			Contact Us </span
+		>)
+	</p>
+</html>
+```
+
+#### Inapplicable Example 5
+
+These links have empty [accessible names][accessible name].
+
+```html
+<a href="http://facebook.com"></a> <a href="http://twitter.com"></a>
+```
+
+#### Inapplicable Example 6
+
+These image links have empty [accessible names][accessible name].
+
+```html
+<a href="http://facebook.com"><img src="facebook.jpg"/></a> <a href="http://twitter.com"><img src="twitter.jpg"/></a>
```

Comment:
As above, I don't think these satisfy the SC.

Reply:
Removed this example, given that it was testing the same as the previous.